README.md

```diff
@@ -545,6 +546,147 @@ module.export = function migrateData(app) {
 }
 ```
 
+# Force Id
```

Comment:
Can we move this part under the model? It's more about the model design:

- Model
         - Map Between Model And Document
         - Map Between Model And Document
         - Model-specific Configuration		        
         - _rev Property
        ... ...
         - Force Id